docs/structures/examples.md

```diff
@@ -0,0 +1,132 @@
+* example.js
+
+#### description.md
+
+This is a Markdown file containing some front matter. The front matter may contain up to three properties:
+
+* `title`: Optional. A human-readable title for this example. If present this will be represented in the JSON and used as a section heading by the MDN renderer. If an item contains more than one example with a title, then all titles must be unique. If any examples for an item have titles, then all of them must (so it's OK for there to be multiple examples with no titles, but not for some to have titles and others not).
+
+* `height`: Optional, but if given, then `width` must also be given. If this is present then the example is treated as a live sample, and this represents the height of the result iframe in pixels.
+
+* `width`: Optional, but if given, then `height` must also be given. If this is present then the example is treated as a live sample, and this represents the width of the result iframe in pixels.
+
+The rest of "description.md" consists of a description of this example.
+
+Example "description.md":
+
+```
+---
+title: Marking up an abbreviation semantically
+width: 672
+height: 192
+---
+To mark up an abbreviation without providing an expansion or
+description, simply use `<abbr>` without any attributes, as seen in this example.
+```
+
+#### example.css, example.html, example.js
+
+These files contain the example's CSS, HTML, and JS. Only one file of each type may be included.
+
+## JSON format
+
+This section describes how the stumptown content will be represented in JSON format. This is the format that would be made available to consumers, including the thing that renders MDN pages.
+
+The item will get a property `examples`. This is a list of examples in the order given in the meta.yaml file.
+
+Each item in the list is an object with the following properties, both optional:
+
+* `description`: if present this is an object which may contain up to four properties, all optional:
+  * `title`: the title from "description.md" front matter
+  * `width`: the width from "description.md" front matter
+  * `height`: the width from "description.md" front matter
+  * `content`: the Markdown from "description.md", converted to HTML
+* `sources`: if present this is an object with the following properties, all optional:
+  * `css`: the content of "example.css"
+  * `html`: the content of "example.html"
+  * `js`: the content of "example.js"
+
+For example:
+
+```json
+"examples": [
+  {
+    "description": {
+      "title": "Simple video example",
+      "width": 640,
+      "height": 370,
+      "content": "<p>This example plays a video...</p>\n"
+    },
+    "sources": {
+      "html": "<video src=...>...</video>\n"
+    }
+  },
+  {
+    "description": {
+      "title": "Multiple video example",
+      "width": 640,
+      "height": 370,
+      "content": "<p>This example...</p>\n"
+    },
+    "sources": {
+      "html": "<video...>...</video>\n"
+    }
+  }
+]
+```
+
+## Rendering
```

Comment:
If we're going to write this up, that's fine, I understand why we're doing it. But I do like making the split between stumptown and consumers explicit. With that in mind, I'd characterize this as "rendering suggestions"

Reply:
You've already written it now but I think the documentation would be easier to maintain and easier to write if we stay away from saying too much about rendering. It would also send a nice signal that the content is just the content.